docs/community/README.md

```diff
@@ -50,13 +50,13 @@ Contributions are welcomed and greatly appreciated. See [CONTRIBUTING.md](contri
 
 ## How Do I Become a Committer?
 
-First of all you need to get involved and be a Contributor.
+First of all, you need to get involved and be a Contributor.
 
-Based on your track-record as a contributor, Per Apache code, PMCs vote on committership, may invite you to be a committer (after we've called a vote). When that happens, if you accept, the following process kicks into place...
+Based on your track-record as a contributor, Per Apache code, PMCs vote on committers, may invite you to be a committer (after we've called a vote). When that happens, if you accept, the following process kicks into place...
```

Comment:
actually, the word "comittership" makes more sense here.

Reply:
Got it